### PR TITLE
fix(1378): vanishing artist series

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

This PR resolves [CX-1378]

### Description
- Fix vanishing artist series

**Before**

https://user-images.githubusercontent.com/11945712/119138191-eacee400-ba41-11eb-896a-864ac63ad36c.mov

**After**

https://user-images.githubusercontent.com/11945712/119138205-edc9d480-ba41-11eb-800e-d27eca1a5ea7.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-1378]: https://artsyproduct.atlassian.net/browse/CX-1378

### CODE REVIEW

File: src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx [hunk 1]
```diff
@@ -33,13 +33,20 @@ export const ArtistSeriesMoreSeries: React.FC<ArtistSeriesMoreSeriesProps> = ({
...
+  const [artistSeries, setArtistSeries] = useState(series)
   const excludedArtistSeriesCount = currentArtistSeriesExcluded ? 1 : 0
   const totalCount = Number(artist?.artistSeriesConnection?.totalCount ?? 0) + excludedArtistSeriesCount
 
   if (!artist || series.length === 0) {
     return null
   }
 
+  // We are saving the artist series to the state here because we face a weird
+  // issue where this list gets updated whenever the user navigates to the ArtistsSeries
```
Comment (MounirDhahri) [inline src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx:45]: the issue seems to be gravity-related, but I didn't think it's a good idea to dig deep in that rabbit hole

Comment (pvinis) [reply src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx:45]: I agree, but I also think this hack is a hacky hack. Could you make a ticket to revert this PR and fix the actual issue once we have shipped android? 🙏

**Review by pvinis — APPROVED** [review]

**Comment by pvinis** [comment]
I'll merge this, but definitely we need to make an issue to revert this PR and fix the actual issue.

File: CHANGELOG.yml [hunk 2]
```diff
@@ -15,12 +15,7 @@ upcoming:
     - Add artwork Material filter - dzmitry tratsiak
     - Prevent android dark mode forced on users - kizito
     - Fix unusable slider on android - dzmitry
-    - Add search UI for material filter - dzmitry tratsiak
-    - Implements custom size filter with localization (behind feature flag) - damon
-    - Adds artist nationality & ethnicity filter (behind feature flag) - damon
-    - Add order history to user profile (behind feature flag) - alexj105
-    - Enable lotsByFollowedArtists - ole
-    - Add ability to unselected "ways to buy" filter - dzmitry tratsiak
+    - Fix vanishing artist series - mounir
```
Comment (pvinis) [inline CHANGELOG.yml:18]:
```suggestion
    - Fix vanishing artist series - mounir
    - Add search UI for material filter - dzmitry tratsiak
    - Implements custom size filter with localization (behind feature flag) - damon
    - Adds artist nationality & ethnicity filter (behind feature flag) - damon
    - Add order history to user profile (behind feature flag) - alexj105
    - Enable lotsByFollowedArtists - ole
    - Add ability to unselected "ways to buy" filter - dzmitry tratsiak
```
OMG I HATE CHANGELOG CONFLICTS

File: src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx:45]: Here is the ticket: https://artsyproduct.atlassian.net/browse/CX-1503
I understand that this is definitely a hack, this is why I added my comment about itit. But it seemed to me that digging into our suggestion engine to fix a disappearing item was not worth it for now but it I am open to that in the future.

File: CHANGELOG.yml (hunk 2 above)
Comment (MounirDhahri) [reply CHANGELOG.yml:18]: soon we won't have to deal with them 🙌

File: src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx (hunk 1 above)
Comment (pvinis) [reply src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx:45]: Yea for sure, better to quickly fix it for now. Thanks for the ticket and the PR!

Comment (MounirDhahri) [reply src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx:45]: of course 🙇‍♂️

**Pull request merged**